src/template-tags/tickets.php

```diff
@@ -123,6 +123,11 @@ function tribe_tickets_display_count_and_ticket_button() {
 		// get an array for ticket and rsvp counts
 		$types = Tribe__Tickets__Tickets::get_ticket_counts( $event_id );
 
+		// if no rsvp or tickets return
+		if ( ! $types ) {
```

Comment:
Is `0 === $types['ticket'] && 0 === $types['rsvp']` a possibility we need to test for?